mozdownload/scraper.py

```diff
@@ -74,7 +74,8 @@
                            'mac': 'dmg',
                            'mac64': 'dmg',
                            'win32': 'exe',
-                           'win64': 'exe'}
+                           'win64': 'exe',
+                           'win64-aarch64': 'exe'}
```

Comment:
Please name the architecture `arm64` similar to what we make use of for Linux and Android. This applies to all the other changes as well.

Reply:
Sorry for the delay but the review re-request just got under the wire. Checking the updated code this issue has been addressed and can be closed. Thanks for making those changes.